src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/netstandard1.0/Microsoft.NET.Sdk.Functions.Build.targets

```diff
@@ -28,13 +28,19 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
       OverwriteReadOnlyFiles="true" 
       Condition="Exists('$(TargetDir)$(TargetName).pdb')"
       ContinueOnError="true"/>
-    
+
+    <ItemGroup>
+      <UserProvidedFunctionJsonFiles Include="@(None)" Condition="Exists(%(None.Identity)) 
```

Comment:
If the included json files are in the Content itemgroup, then it could get missed 

```
    <Content Include="FooFunction\FooFunction.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
```

Reply:
how do you fix that? I can merge the PR and just fix this issue myself if possible?

Reply:
I wanted to keep the considered files minimal. I think the default build action is `None `(when adding a `function.json` via Visual Studio).

But I think including `Content` items is a good idea to avoid confusion. Should we also include `Copy if newer`?

Reply:
I added a commit to include "Content" items (just in case this approach is used). Would you please review @vijayrkn @ahmelsayed? Thanks!

Reply:
@cschwendtner - As you mentioned, I think it will be a good idea to include PreserveNewest as well since it is configurable from the VS UI.

Reply:
@vijayrkn I added a commit to respect items with `PreserveNewest`. 

But there is a corner case: 
- the user adds a function.json file and sets `CopyToOutputDirectory` to `PreserveNewest`
- the user compiles (custom function.json is used / ignored by the generator)
- the user sets `CopyToOutputDirectory` to `Never` (to use the generator)
- the user compiles (function.json is generated by the generator)
- the user sets `CopyToOutputDirectory` to `PreserveNewest` to (try to) use the custom function.json file again

In this cases (and some variations) the custom function.json file is not copied to the output directory because the file generated by the generator is newer.